resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -212,7 +212,9 @@ class KubernetesSuite extends SparkFunSuite
       driverPodChecker: Pod => Unit = doBasicDriverPodCheck,
       executorPodChecker: Pod => Unit = doBasicExecutorPodCheck,
       appArgs: Array[String] = Array.empty[String],
-      isJVM: Boolean = true ): Unit = {
+      isJVM: Boolean = true,
+      sparkConfOverride: Option[SparkAppConf] = None,
+      appLocatorOverride: Option[String] = None): Unit = {
```

Comment:
Hi, I don't understand what word `override` means. Could you explain it?

Reply:
I understand. I don't think it's best to name it as `override`. How about `specifiedSparkConf` ?

Reply:
I was the one who suggested `sparkConfOverride`. I agree that it might be confusing. Here is another alternative - `customSparkConf` ?

Reply:
@martin-g Thanks for your sharing. I also agree that `customSparkConf` is a good choice.

Reply:
Thanks, done